pkg/graphql/src/schema/gameDao.graphql

```diff
@@ -26,6 +26,17 @@ interface ProposalFeatures {
     CREATE_WITHDRAW_PROPOSAL: Boolean!
     CREATE_SPENDING_PROPOSAL: Boolean!
 }
+type OrganizationPageFeaturesSortOptions {
+    name: String!
+    value: String!
+}
+
+type OrganizationPageFeatures {
```

Comment:
No nesting use an interface. Look at ProposalFeatures

Reply:
I think no need for that since it is now only three booleans